docker/dev/Dockerfile

```diff
@@ -192,7 +202,16 @@ RUN $CMAKE --build $BASE_PATH/parflow/build
 
 USER root
 RUN $CMAKE --install $BASE_PATH/parflow/build --prefix /opt/parflow
+
+# Install the locally built version of pftools
 USER ubuntu
+RUN python3 -m pip install /home/ubuntu/parflow/build/pftools/python
```

Comment:
This fixes a bug where the install of pftools for python was never being completed